### ISSUE TITLE
Enabling Limited Federation Mode breaks /about page.

### ISSUE THREAD

**Issue by ztaylor100** [mastodon/mastodon#17090]
### Steps to reproduce the problem

1. Enable Limited Federation Mode
2. Click on the "About this server " link
3. Arrive at "The page you are looking for isn't here." error page
4. URL requested is https://yourdomain.com/about/more
5. This behavior is occurring while authenticated 
...

### Expected behaviour

Should return about page

### Actual behaviour

Returns error page

### Specifications

Ubuntu 18.04 Server LTS
Mastodon 3.4.4
Ruby 2.7.2p137
PostgreSQL 10.19
Redis 4.0.9

**Comment by ClearlyClaire** [comment]
This is per @Gargron's design: all “public” pages are disabled on purpose in limited federation mode, including `/about/more`

**Comment by ztaylor100** [comment]
Ok, thank you for the clarification. I was confused because all of the other "links" work such as terms of service and my preference is always to not have my users think something is broken.

**Comment by ztaylor100** [comment]
Would it be terribly difficult to allow access to the about and directory pages with an if-authenticated statement?

**Comment by ClearlyClaire** [comment]
No, it wouldn't, and I personally don't think we should disable `/about/more` at all, but that was @Gargron's decision

**Comment by ztaylor100** [comment]
Hello @Gargron! I think it would be fantastic to have the ability as an administrator to be able to decide if we want to enable the /about and /about/more pages as well as the profile directory (so that people can actually find each other) for those of us running limited federation instances. Just my personal humble opinion. Any chance of this happening in the future?

**Comment by jcravener** [comment]
I agree.  I think the /about/more page should be available in Limited Federation mode.  I encountered this error too but only after enabling AUTHORIZED_FETCH.  In fact the /terms page is still available in this Limited Federation mode.

**Comment by ztaylor100** [comment]
@Gargron can you please explain the use case for Limited Federation Mode with people wanting a private instance, but nobody can find each other because the directory is disabled? How about giving us admins some buttons to turn things like this on and off at our own discretion?
